src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -0,0 +1,37 @@
+import { LinkProps } from "components/link/link.component";
+import React from "react";
+import tagComponent, {
+  TagProps,
+} from "../../../__internal__/utils/helpers/tags";
+import { StyledCrumb, Divider } from "./crumb.style";
+
+export interface CrumbProps
+  extends Omit<LinkProps, "toolTipMessage" | "tooltipPosition" | "iconType">,
+    TagProps {
+  /** This sets the Crumb to current, does not render Link */
+  isCurrent?: boolean;
+  /** The href string for Crumb Link */
+  href: string;
+}
+
+const Crumb = React.forwardRef(
+  ({ href, isCurrent, children, ...rest }: CrumbProps) => (
+    <li>
+      <StyledCrumb
+        isCurrent={isCurrent}
+        aria-current={isCurrent}
```

Comment:
Taking another look at `aria-current`, I think we need to [set it to `"page"` similar to this example.](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#example)  The screen reader would then announce that link refers to the name of the page the user is on. (It seems `aria-current` can be used in other contexts as well)

```suggestion
        aria-current={isCurrent ? "page" : undefined}
```

Reply:
`location` could be equally valid. There's some talk about it in the comments of [this blog post](https://tink.uk/using-the-aria-current-attribute/).